admin/data/sql/migrations/20191011004727.do.create-service-message-table.sql

```diff
@@ -0,0 +1,29 @@
+CREATE TABLE [mtc_admin].[serviceMessage] (
+  id int IDENTITY (1,1) NOT NULL,
+  createdAt datetimeoffset(3) NOT NULL DEFAULT GETUTCDATE(),
+  updatedAt datetimeoffset(3) NOT NULL DEFAULT GETUTCDATE(),
+  version rowversion,
+  recordedByUser_id int NOT NULL,
```

Comment:
To keep this consistent with the `createdAt` & `updatedAt` DateTime fields shouldn't we call this field `createdByUser_id`?

Reply:
@jon-shipley updated.